### PR TITLE
Style inline images

### PR DESCRIPTION
**Pull request by caffo**
yyccnk4e

### CODE REVIEW

File: src/styles/image.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+ ==============================================================================
+ Images
+ ==============================================================================
+*/
+
+.figure a {
+      background: none !important;
```
Comment (rafaelrinaldi) [inline src/styles/image.css]: @caffo Bad indenting.

File: src/styles/index.css [hunk 2]
```diff
@@ -23,6 +23,7 @@
 @import './selection';
 @import './strike.css';
 @import './table.css';
+@import './image.css';
```
Comment (rafaelrinaldi) [inline src/styles/index.css]: @caffo Maybe rename it to `inline-image`?

File: src/styles/image.css (hunk 1 above)
Comment (caffo) [inline src/styles/image.css]: My bad. 🍭

File: src/styles/index.css (hunk 2 above)
Comment (caffo) [inline src/styles/index.css]: I can't imagine any other image type that would need styles. Any reason to not generalize? 🙇

Comment (rafaelrinaldi) [inline src/styles/index.css]: @caffo Alright, ship it.

**Pull request merged**